cmd/run.go

```diff
@@ -114,12 +114,18 @@ func runCommand(cmd *cobra.Command, args []string) {
 	// Output our results
 	log.Infof(resultFormat, "Instance ID", "Region", "Profile", "Status")
 	for _, v := range output.InvocationResults {
-		switch v.Status {
```

Comment:
Is there a particular reason to move away from the switch/case statements?

Reply:
Jordan and I paired on this. Not sure why I made the change either